android/src/main/java/com/reactnativestripesdk/PaymentMethodCreateParamsFactory.kt

```diff
@@ -99,14 +110,14 @@ class PaymentMethodCreateParamsFactory(private val clientSecret: String, private
 
     if (paymentMethodId != null) {
       val cvc = getValOr(params, "cvc", null)
-      val paymentMethodOptionParams = if (cvc != null) PaymentMethodOptionsParams.Card(cvc) else null
```

Comment:
We should also pass appID to paymentOptions when it's wechat by
```
val paymentMethodOptionParams = PaymentMethodOptionsParams.WeChatPay(appId)
```
like what we are doing in iOS `PaymentMethodFactory.createWeChatPayPaymentMethodOptions`